content/en/docs/contribute/style/content-guide.md

```diff
@@ -10,107 +10,68 @@ card:
 ---
 
 {{% capture overview %}}
-This page contains guidelines for adding content to the Kubernetes documentation.
-If you have questions about allowed content, join the [Kubernetes Slack](http://slack.k8s.io/) #sig-docs channel and ask! Use your best judgment, and feel free to
-propose changes to this document in a pull request.
 
-For additional information on creating new content for the Kubernetes
-docs, follow the instructions in the [Style guide](/docs/contribute/style/style-guide).
+This page contains guidelines for Kubernetes documentation.
+
+If you have questions about what's allowed, join the #sig-docs channel in 
+[Kubernetes Slack](http://slack.k8s.io/) and ask! 
+
+You can register for Kubernetes Slack at http://slack.k8s.io/. 
+
+For information on creating new content for the Kubernetes
+docs, follow the [style guide](/docs/contribute/style/style-guide).
+
 {{% /capture %}}
 
 {{% capture body %}}
-## Contributing content
 
-The Kubernetes documentation comprises the content of the
-[kubernetes/website](https://github.com/kubernetes/website) source repository.
+## Overview
```

Comment:
@zacharysarah I think this could move to the overview section (that starts on line 12). I don't feel strongly about this, so if you prefer things as they are then no worries.